src/components/RegisterForm.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "../style/AccountForm.css";
+import { login, register } from "../axios-services"; 
+
+const RegisterForm = ({ setToken }) => {
+  const [full_name, setFull_Name] = useState("");
+  const [email, setEmail] = useState("");
+  const [username, setUserName] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
you can follow add product component from admin as example so you don't have to keep multiple useState lines, this way you have a single user object  that has the different properties that you need to change